### DISCUSSION TITLE
Settings Ladesteuerung

### DISCUSSION THREAD

**Discussion by aidaodeon** [wiggal/GEN24_Ladesteuerung#72] [General]
Hallo,
danke für die Ladesteuerung, funktioniert ganz gut und ist eine echte Hilfe.
Im Moment nutze ich die HTTP-Ladesteuerung
Ich habe noch ein paar Verständnisprobleme mit den Einstellungen „Settings“.

-	bei der Einstellung „WR-Steuerung und Logging“ werden die Werte auf den Wechselrichter geschrieben.
-	Bei der Einstellung „AUS“ sind die Scripte deaktiviert.

Bei „AUS“ wird aber weiter mit den letzten Werten aus dem Log geladen. Nach meinem Verständnis sollte aber mit aller verfügbaren Energie geladen werden da der Wechselrichter nicht mehr angesteuert wird.
So hat diese Einstellung keine Auswirkung auf den Ladewert. Vielleicht verstehe ich auch was falsch.
Bitte um Hilfe.
Danke
Herberth

**Comment by wiggal** [comment]
>     * bei der Einstellung „WR-Steuerung und Logging“ werden die Werte auf den Wechselrichter geschrieben.

Ja die Werte werden auf den Wechselrichter geschrieben und die Daten in die SQLite-Datei.
> 
>     * Bei der Einstellung „AUS“ sind die Scripte deaktiviert.
> Bei „AUS“ wird aber weiter mit den letzten Werten aus dem Log geladen. Nach meinem Verständnis sollte aber mit aller verfügbaren Energie geladen werden da der Wechselrichter nicht mehr angesteuert wird. So hat diese Einstellung keine Auswirkung auf den Ladewert. Vielleicht verstehe ich auch was falsch. Bitte um Hilfe. Danke Herberth

ja, ist die Bezeichnung "AUS" wohl etwas irreführend. 
Die Alternative, die Werte der Steuerung ganz zu deaktivieren, ist mir noch gar nicht in den Sinn gekommen. 
"AUS" bedeutet in dem Fall, dass die Steuerung nichts mehr macht, auch nicht die alten Ladewerte deaktivieren.
Eventuell wäre "Pausieren" die bessere Bezeichnung?

**Reply by aidaodeon** [reply]
Hallo,
danke für die Info, die Steuerung verhält sich so wie du es beschreibst.
Wie kann ich die alten Ladewerte deaktivieren damit der Wechselrichter mit den originalen Einstellungen läuft?
In manchen Situationen (Pool, Wäsche, Geschirrspüler) wenn es am Nachmittag bewölkt ist wird die Batterie nicht mehr voll, im Gegenteil sie wird entladen. Ich speise aber den Strom vorher ein. In diesem Fall ist es sinnvoll die Batterie vorher voll zu laden und nach dem entladen mit vollen Ladewerten zu laden.
Danke
Herberth

**Reply by wiggal** [reply]
kommt drauf an, welche Version di vewendest.
Bei HTTP einfach im Batteriemanagement deaktivieren odr löschen.
Bei Modbus ausschalten, sichern wieder einschalten, sichern

**Reply by wiggal** [reply]
Prinzipiell gebe ich dir Recht, wenn der Schalter auf "AUS" ist, sollten die bisherigen Einstellungen der Steuerung deaktiviert werden.
Ich versuche es irgendwann einzubauen, und in der Hilfe zu beschreiben!

**Reply by wiggal** [reply]
@aidaodeon 
welche Version verwendest du?

**Reply by aidaodeon** [reply]
Danke für deine Hilfe.
Jetzt vestehe ich warum ich die Einstellungen nicht deaktivieren konnte. Ich weiß jetzte dank deiner Erläuterungen wie ich mir helfen kann.
Ich habe jetzt die Version 21.0 installiert.

Andere Baustelle: die Aktualisierung mache ich mit mit "git pull". Bei Änderung der Dateien, z. B. config.ini, meldet das Programm das ich vor merge, comitten oder stashen soll. Damit komme ich überhaupt nicht klar.
Kannst du mir bitte die Befehle für die Aktualisierung mitteilen?

Danke für dein Engajement und für deine Hilfe.

**Reply by wiggal** [reply]
Hi,
da hab ich mich nicht richtig ausgedrückt, mit Version meinte ich Modbus oder HTTP.

Zu git:
in deinem lokalen Verzeichnis sind ja Dateien gegenüber von Github verändert, du musst die veränderten Dateien vorher 'commiten'.
git status        # zeigt dir alle veränderten Dateien
git add --all    # stellt alle veränderten Dateien zum commit bereit
git commit -m 'Erläuterung zum Commit'  # bindet die veränderten Dateien lokal ein.
git pull        # müsste nun die veränderten Dateien von Github holen (außer deine lokal veränderten)

Wenn du das probierst, mach vorsichtshalber eine Sicherungskopie von deinem Verzeichnis (ich kann das so nicht probieren, da ich die neuesten Dateien auf dem Entwicklerrechner habe)

**Reply by aidaodeon** [reply]
Danke.
Ic nutze HTTP.

**Reply by wiggal** [reply]
Hab die Deaktivierung der Ladewerte unter dem Punt AUS nun eingebaut.
